[#3] Add Axon Server section (#8)
AxonFramework/.github: profile/README.md

* **[Axon Framework](#axon-framework)** 
* **[Getting Started](#getting-started)** 
* **[Receiving Help](#receiving-help)**

> * **[Axon Server](#axon-server)** - Purpose built Message router for commands, event, and queries, and an efficient Event Store 

Reviewer: ```suggestion
* **[Axon Server](#axon-server)** - Purpose-built Event Store and Messaging Platform for commands and queries
```

Author: Very good point. Replaced my text with yours.

The message-driven nature of the framework becomes particularly useful when scaling the application or breaking it apart into distinct [microservice](https://developer.axoniq.io/microservices/overview).
Furthermore, by Event Sourcing, you are future-proof to any shift and change of the models.

Both benefits come with their own set of requirements.
Firstly, we are inclined to use a distributed version of the `CommandBus`, `EventBus`, and `QueryBus` to break apart an application, considering their distinct routing requirements.
Secondly, as Event Sourcing requires us to store events forever, the `EventStore` will grow indefinitely.

[Axon Server](https://developer.axoniq.io/axon-server/overview) greatly simplifies these requirements by implementing these four infrastructure components.
It is a dedicated message routing solution knowledgeable about the need to consistently route commands, stream events as fast as possible, and support the differing querying needs of each service.

> Next to message routing, Axon Server contains a storage engine designed for the specific purpose of storing events for event sourcing.

Reviewer: ```suggestion
But most of all, Axon Server is an event store, a database specifically designed for storing events for event sourcing purpose.
```

Note that although you can use this extension to adjust MongoDB into an Event Store, we do not necessarily recommend this.
During the lifecycle of this extension, we have noticed predicaments with how MongoDB behaves, causing the Event Store to act suboptimal from a performance perspective.
We thus recommend either [Axon Server](#axon-server) (the highest level of performance for event storage) or an RDBMS of choice instead.

> 

Reviewer: I believe it is better to write event store in lowercase.

Author: Makes sense, will do!!
